lib/modules/dosomething/dosomething_mbp/dosomething_mbp.install

```diff
@@ -108,3 +108,45 @@ function dosomething_mbp_update_7003(&$sandbox) {
   }
 
 }
+
+/**
+ * Switch affiliate sites to use Message Broker for transactional email messages.
+ */
+ function dosomething_mbp_update_7004(&$sandbox) {
+
+   $modules = array(
+     'message_broker_producer',
+     'dosomething_mbp'
+   );
+   module_enable($modules);
+
+   // Default Message Broker RabbitMQ connection settings
+   variable_set('message_broker_producer_rabbitmq_host', '10.241.0.14');
+   variable_set('message_broker_producer_rabbitmq_port', '5672');
+   variable_set('message_broker_producer_rabbitmq_username', 'dosomething');
+   variable_set('message_broker_producer_rabbitmq_vhost', 'dosomething');
+
+   $affiliate = dosomething_settings_get_affiliate_country_code();
+   $affiliate_lc = strtolower($affiliate);
+
+   if ($affiliate == 'US') {
+     variable_set('dosomething_mbp_user_register_template', 'mb-general-signup-us');
+     variable_set('dosomething_mbp_user_password_template', 'mb-password-reset-us');
+     variable_set('dosomething_mbp_campaign_signup_template', 'mb-campaign-signup-us');
+     variable_set('dosomething_mbp_campaign_reportback_template', 'mb-campaign-report-back-us');
```

Comment:
To keep this consistent within the codebase, I'd vote that we name this template as `reportback` and not `report-back` if possible